tests/examples/start-message/playground/start-message/start-msg

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+# Not actually, but to demonstrate
+echo "Service has started at localhost:9001" 1>& 2
```

Comment:
ah derp, originally had `2>& 1` and realized that was backwards and switched it to `2>& 1` :P